crates/astria-composer/src/main.rs

```diff
@@ -18,10 +18,18 @@ async fn main() -> ExitCode {
         }
     };
 
-    if let Err(e) = telemetry::configure()
+    let mut telemetry_conf = telemetry::configure()
         .set_no_otel(cfg.no_otel)
         .set_force_stdout(cfg.force_stdout)
-        .filter_directives(&cfg.log)
+        .filter_directives(&cfg.log);
+
+    if cfg.metrics_enabled {
+        telemetry_conf = telemetry_conf
+            .metrics_addr(&cfg.prometheus_http_listener_addr)
+            .service_name(env!("CARGO_CRATE_NAME"));
```

Comment:
Should this be configurable so that operators can inject their own global service name to identify metrics?